pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -278,6 +278,8 @@ public void handle(Request request, Response response) throws Exception {
         // - StudyLanguageContentLanguageSettingFilter sets the "Content-Language" header later on
         before(API.BASE + "/user/*/studies/*", new StudyLanguageResolutionFilter());
         after(API.BASE + "/user/*/studies/*", new StudyLanguageContentLanguageSettingFilter());
+        before(API.BASE + "/studies/*", new StudyLanguageResolutionFilter());
+        after(API.BASE + "/studies/*", new StudyLanguageContentLanguageSettingFilter());
```

Comment:
We have to mount filters 2 times because `*/studies/*` doesn't work. `*studies/`, on the other hand, seems to be too permissive. So far the only route falling under `*/studies/*` is `GetStudyDetailRoute`.